zeppelin-web/src/app/spell/spell-base.js

```diff
@@ -31,9 +31,10 @@ export class SpellBase {
    * Consumes text and return `SpellResult`.
    *
    * @param paragraphText {string} which doesn't include magic
+   * @param config {Object}
    * @return {SpellResult}
    */
-  interpret(paragraphText) {
+  interpret(paragraphText, config) {
```

Comment:
Curious idea behind passing 'config' to interpret(), instead of passing it to constructor. Could you elaborate?